tests/other/metadce/minimal_main_Oz_USE_PTHREADS_PROXY_TO_PTHREAD.jssize

```diff
@@ -1 +1 @@
-47521
```

Comment:
This is a pretty nice size reduction!

Reply:
I managed to reduce it even more with commit b3beb21d7d1bc0fdaf5eff6914a11501332952b1.